README.md

```diff
@@ -1,47 +1,79 @@
 # Generate SBOM GitHub Action
```

Comment:
I think we would want a full example that scan ISO + repo and generate a resulting archive + upload it (just so that it will be simpler for others to use it the same way we do)